accounting/invoice/tests/test_admin.py

```diff
@@ -118,18 +130,29 @@ def test_mark_paid(self):
         self.assertFalse(self.invoice3.paid)
 
     def test_mark_approved(self):
-        """Marking a set of invoices as approved through the admin should only mark selected invoices as approved.
-        Approval status remains the same if already previously approved."""
+        """Marking a set of invoices as approved through the admin should only mark selected invoices as manually
+        approved. Approval status remains the same if already previously approved."""
+        # An extra, unapproved invoice to leave out of selection, to ensure it isn't approved after the operation.
+        invoice4 = factories.InvoiceFactory(
+            client=self.client_account,
+            provider=self.provider1,
+            pdf_path='https://drive.google.com/f/asdf4.pdf',
+            paid=True,
+            approved=choices.InvoiceApproval.not_approved
+        )
         self.client.force_login(self.admin_user)
         self.client.post(self.url, data={
             'action': 'mark_approved',
             ACTION_CHECKBOX_NAME: [self.invoice1.pk, self.invoice2.pk, self.invoice3.pk],
         })
 
         self._refresh_invoices()
+        invoice4.refresh_from_db()
         self.assertTrue(self.invoice1.is_approved)
         self.assertTrue(self.invoice2.is_approved)
         self.assertTrue(self.invoice3.is_approved)
-        self.assertTrue(self.invoice1.approved, choices.InvoiceApproval.manually)
-        self.assertTrue(self.invoice2.approved, choices.InvoiceApproval.manually)
-        self.assertTrue(self.invoice2.approved, choices.InvoiceApproval.automatically)
+        self.assertFalse(invoice4.is_approved)
+        self.assertEqual(self.invoice1.approved, choices.InvoiceApproval.manually)
+        self.assertEqual(self.invoice2.approved, choices.InvoiceApproval.manually)
+        self.assertEqual(self.invoice3.approved, choices.InvoiceApproval.automatically)
```

Comment:
@UmanShahzad How did this invoice get `automatically` approved? That's what I was trying to get at with the second suggestion from https://github.com/open-craft/accounting/pull/14#discussion_r164689448 :)

Reply:
@itsjeyd Oh, it's already set that way in the set up function.